packages/salesforcedx-vscode-core/src/orgBrowser/index.ts

```diff
@@ -8,3 +8,4 @@ export { TypeUtils, MetadataObject } from './metadataType';
 export { MetadataOutlineProvider } from './metadataOutlineProvider';
 export { BrowserNode, NodeType } from './nodeTypes';
 export { ComponentUtils } from './metadataCmp';
+export { OrgBrowser } from './browser';
```

Comment:
I think you can replace this with something like:
```
export const orgBrowser = OrgBrowser.get();
```
That way you don't have to do `OrgBrowser.get()` every time you import it in other places in the codebase e.g. line 408 in `packages/salesforcedx-vscode-core/src/index.ts`